src/duplo_resource/user.py

```diff
@@ -41,3 +41,10 @@ def add_user_to_tenant(self,
       raise DuploError(f"Failed to add user '{name}' to tenant '{tenant}'", res["status_code"])
     else:
       return f"User '{name}' added to tenant '{tenant}'"
+    
+  @Command()
+  def create(self, 
```

Comment:
I see what's up with the `state` key in the body now. 
You can instead always make sure `state` is `add` in `create`. 
Like: `body['state'] = 'add'` 
Then make a delete method which only takes the name and make the body with `state` as `delete`. This way we ensure the common CRUD methods are present.